packages/gitbook/src/components/DocumentView/Caption.tsx

```diff
@@ -6,7 +6,7 @@ import {
     JSONDocument,
 } from '@gitbook/api';
 
-import { getNodeFragmentByName, isNodeEmpty } from '@/lib/document';
+import { getNodeFragmentByName, isNodeEmpty } from '@/lib/document-util';
```

Comment:
Why rename this file? it's already in `lib` so `util` is kind of obvious

Reply:
It was just during the refactoring, can rename it.